src/agent/debuglet.js

```diff
@@ -23,10 +23,10 @@ var util = require('util');
 var semver = require('semver');
 
 var v8debugapi = require('./v8debugapi.js');
-var DebugletApi = require('../debugletapi.js');
+var DebugletApi = require('../controller.js');
```

Comment:
The variable name is being fixed in a follow-on change. The API class name was changed to match the official API description: https://cloud.google.com/debugger/api/reference/rest/.